.eslintrc

```diff
@@ -14,7 +15,7 @@
         "indent": [2, 4, {"SwitchCase": 1}],
         "keyword-spacing": 2,
         "linebreak-style": [2, "unix"],
-        "max-len": [2, 80, 4, { "ignoreUrls": true, "ignorePattern": "\\brequire\\([\"']|eslint-disable" }],
+        "max-len": [2, 90, 4, { "ignoreUrls": true, "ignorePattern": "\\brequire\\([\"']|eslint-disable" }],
```

Comment:
@kevinbarabash Adding classes adds about 4 spaces to each line.  most of the Parser code about 13 lines have error. I think it is a lot. I decrease it to 84 but less than that makes a lot of lint errors